### DISCUSSION TITLE
The random function does not work

### DISCUSSION THREAD

**Discussion by Lindsaydoodle** [Dhruv-Techapps/acf-docs#362] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/45480043/160237062-2fa25efe-3c91-4ae5-9ab4-f3302294e7d9.png)
![image](https://user-images.githubusercontent.com/45480043/160237067-8113a297-3b76-4069-a0d6-efc3e479787d.png)
![image](https://user-images.githubusercontent.com/45480043/160237070-a1dc4451-8299-46b7-812f-b94bafa955a9.png)
![image](https://user-images.githubusercontent.com/45480043/160237074-1d76fffc-ca93-4b72-b6d3-0daa140a4f3e.png)

I don't know what I'm doing wrong and why it doesn't randomize but just inserts the code. According to the documentation I do everything as it is written. Please help me

**Comment by nXwhite** [comment]
Are you sure that there is an input-field where you try to put in the random code?

I've tested it on "google"-search-line with "<random[a-z]{6}>" and it worked perfectly!

**Comment by nXwhite** [comment, marked as answer]
Forget my first question! I looked again on your screenshots and now i can see, that there is definitely a "input field". So it looks like you have done everything right! 
But do you use the developer version of the extension? I'm pretty sure the random-stuff only works in the DEV Version!
